tests/test_resolver.py

```diff
@@ -1260,92 +1264,6 @@ def test_expand_state_intersection_adjust(
             "https://pypi.org/simple",
         ) not in state1.iter_unresolved_dependencies()
 
-    def test_expand_state_no_unresolved(self, resolver: Resolver, state: State) -> None:
```

Comment:
why is this removed?

Reply:
There is guaranteed that no unresolved dependencies (`state.unresolved_dependencies`) are handled in `Resolver._expand_state` (callee) based on the newly introduced logic. This part is a leftover from previous imlementation when we asked for marker evaluation results explictly in `Resolver._expand_state_add_dependecies`. The same applies for the second test removed.